src/object/obj_class.c

```diff
@@ -58,7 +68,176 @@ static struct daos_obj_class daos_obj_classes[] = {
+	{
+		.oc_name	= "S8",
+		.oc_id		= OC_S8,
+		{
+			.ca_schema		= DAOS_OS_STRIPED,
+			.ca_resil		= DAOS_RES_REPL,
+			.ca_grp_nr		= 8,
+			.ca_rp_nr		= 1,
+		},
+	},
+	{
+		.oc_name	= "S16",
+		.oc_id		= OC_S16,
+		{
+			.ca_schema		= DAOS_OS_STRIPED,
+			.ca_resil		= DAOS_RES_REPL,
+			.ca_grp_nr		= 16,
+			.ca_rp_nr		= 1,
+		},
+	},
+	{
+		.oc_name	= "S32",
+		.oc_id		= OC_S32,
+		{
+			.ca_schema		= DAOS_OS_STRIPED,
+			.ca_resil		= DAOS_RES_REPL,
+			.ca_grp_nr		= 32,
+			.ca_rp_nr		= 1,
+		},
+	},
+	{
+		.oc_name	= "S48",
+		.oc_id		= OC_S48,
+		{
+			.ca_schema		= DAOS_OS_STRIPED,
+			.ca_resil		= DAOS_RES_REPL,
+			.ca_grp_nr		= 48,
+			.ca_rp_nr		= 1,
+		},
+	},
+	{
+		.oc_name	= "S64",
+		.oc_id		= OC_S64,
+		{
+			.ca_schema		= DAOS_OS_STRIPED,
+			.ca_resil		= DAOS_RES_REPL,
+			.ca_grp_nr		= 64,
+			.ca_rp_nr		= 1,
+		},
+	},
+	{
+		.oc_name	= "S96",
+		.oc_id		= OC_S96,
+		{
+			.ca_schema		= DAOS_OS_STRIPED,
+			.ca_resil		= DAOS_RES_REPL,
+			.ca_grp_nr		= 96,
+			.ca_rp_nr		= 1,
+		},
+	},
+	{
+		.oc_name	= "S128",
+		.oc_id		= OC_S128,
+		{
+			.ca_schema		= DAOS_OS_STRIPED,
+			.ca_resil		= DAOS_RES_REPL,
+			.ca_grp_nr		= 128,
+			.ca_rp_nr		= 1,
+		},
+	},
+	{
+		.oc_name	= "S192",
+		.oc_id		= OC_S192,
+		{
+			.ca_schema		= DAOS_OS_STRIPED,
+			.ca_resil		= DAOS_RES_REPL,
+			.ca_grp_nr		= 192,
+			.ca_rp_nr		= 1,
+		},
+	},
+	{
+		.oc_name	= "S256",
+		.oc_id		= OC_S256,
+		{
+			.ca_schema		= DAOS_OS_STRIPED,
+			.ca_resil		= DAOS_RES_REPL,
+			.ca_grp_nr		= 256,
+			.ca_rp_nr		= 1,
+		},
+	},
+	{
+		.oc_name	= "S384",
+		.oc_id		= OC_S384,
+		{
+			.ca_schema		= DAOS_OS_STRIPED,
+			.ca_resil		= DAOS_RES_REPL,
+			.ca_grp_nr		= 384,
+			.ca_rp_nr		= 1,
+		},
+	},
+	{
+		.oc_name	= "S512",
+		.oc_id		= OC_S512,
+		{
+			.ca_schema		= DAOS_OS_STRIPED,
+			.ca_resil		= DAOS_RES_REPL,
+			.ca_grp_nr		= 512,
+			.ca_rp_nr		= 1,
+		},
+	},
+	{
+		.oc_name	= "S768",
+		.oc_id		= OC_S768,
+		{
+			.ca_schema		= DAOS_OS_STRIPED,
+			.ca_resil		= DAOS_RES_REPL,
+			.ca_grp_nr		= 768,
+			.ca_rp_nr		= 1,
+		},
+	},
+	{
+		.oc_name	= "S1K",
+		.oc_id		= OC_S1K,
+		{
+			.ca_schema		= DAOS_OS_STRIPED,
+			.ca_resil		= DAOS_RES_REPL,
+			.ca_grp_nr		= (1 << 10),
+			.ca_rp_nr		= 1,
+		},
+	},
+	{
+		.oc_name	= "S2K",
+		.oc_id		= OC_S2K,
+		{
+			.ca_schema		= DAOS_OS_STRIPED,
+			.ca_resil		= DAOS_RES_REPL,
+			.ca_grp_nr		= (2 << 10),
+			.ca_rp_nr		= 1,
+		},
+	},
+	{
+		.oc_name	= "S4K",
+		.oc_id		= OC_S4K,
+		{
+			.ca_schema		= DAOS_OS_STRIPED,
+			.ca_resil		= DAOS_RES_REPL,
+			.ca_grp_nr		= (4 << 10),
+			.ca_rp_nr		= 1,
+		},
+	},
+	{
+		.oc_name	= "S6K",
+		.oc_id		= OC_S6K,
+		{
+			.ca_schema		= DAOS_OS_STRIPED,
+			.ca_resil		= DAOS_RES_REPL,
+			.ca_grp_nr		= (6 << 10),
+			.ca_rp_nr		= 1,
+		},
+	},
+	{
+		.oc_name	= "S8K",
+		.oc_id		= OC_S8K,
+		{
+			.ca_schema		= DAOS_OS_STRIPED,
+			.ca_resil		= DAOS_RES_REPL,
+			.ca_grp_nr		= (8 << 10),
+			.ca_rp_nr		= 1,
+		},
+	},
 	{
 		.oc_name	= "SX",
```

Comment:
Should we still keep SX for now?

Reply:
Yes, we should, user uses it to indicate it is a "large" object.